azure-pipelines-official.yml

```diff
@@ -26,7 +26,7 @@ stages:
     displayName: Official Build
 
     steps:
-    - task: ms-vseng.MicroBuildTasks.30666190-6959-11e5-9f96-f56098202fef.MicroBuildSigningPlugin@1
+    - task: ms-vseng.MicroBuildTasks.30666190-6959-11e5-9f96-f56098202fef.MicroBuildSigningPlugin@3
```

Comment:
Curious to understand why this fixes things. We have other shipping products in dotnet that use [this same version](https://github.com/dotnet/msbuild/blob/a2490dd3f78cce4abc8f9e6f1b5268437332818f/.vsts-dotnet.yml#L90)